doc/release-notes.md

```diff
@@ -64,61 +64,215 @@ platform.
+  progress of the wallet's scanning historical blocks for transactions
+  affecting its balances. (#15730)
+
+- `createwallet` accepts a new `passphrase` parameter.  If set, this
+  will create the new wallet encrypted with the given passphrase.  If
+  unset or empty (the default), a warning will be printed and no
+  encryption will be used. (#16394)
+
+- `getmempoolentry` now provides a `weight` field containing the
+  transaction weight as defined in BIP141. (#16647)
+
+- `getdescriptorinfo` now returns an additional `checksum` field
+  containing the checksum for the unmodified descriptor provided by the
+  user (that is, before the descriptor is normalized for the
+  `descriptor` field). (#15986)
+
+- `walletcreatefundedpsbt` now signals BIP125 Replace-by-Fee if the
+  `-walletrbf` configuration option is set to true. (#15911)
+
+GUI changes
+-----------
+
+- Provides bech32 addresses by default.  The user may change the address
+  during invoice generation using a GUI toggle, or the default address
+  type may be changed by the `-addresstype` configuration option.
+  (#15711, #16497)
+
+Deprecated or removed configuration options
+-------------------------------------------
+
+- `-mempoolreplacement` is removed, although default node behavior
+  remains the same.  This option previously allowed the user to prevent
+  the node from accepting or relaying BIP125 transaction replacements.
+  This is different from the remaining configuration option
+  `-walletrbf`. (#16171)
 
 Deprecated or removed RPCs
 --------------------------
 
-- The `totalFee` option of the `bumpfee` RPC has been deprecated and will be
-  removed in 0.20.  To continue using this option start with
-  `-deprecatedrpc=totalFee`.  See the `bumpfee` RPC help text for more details.
+- `bumpfee` no longer accepts a `totalFee` option unless the
+  configuration parameter `deprecatedrpc=totalFee` is specified.  This
+  parameter will be fully removed in a subsequent release. (#15996)
+
+- `generate` is now removed after being deprecated in Bitcoin Core 0.18.
+  Use the `generatetoaddress` RPC instead. (#15492)
+
+P2P changes
+-----------
+
+- BIP 61 reject messages were deprecated in v0.18. They are now disabled
+  by default, but can be enabled by setting the `-enablebip61` command
+  line option.  BIP 61 reject messages will be removed entirely in a
+  future version of Bitcoin Core. (#14054)
+
+- The default value for the `-peerbloomfilters` configuration option
+  (and, thus, BIP37/BIP111 NODE_BLOOM support) has been changed to false.
+  This eliminates well-known DoS vectors in Bitcoin Core, especially for
+  nodes with spinning disks. Other software should still be able to
+  connect to older versions of Bitcoin Core and users who have manually
+  enabled NODE_BLOOM support for the near future, but developers of such
+  software should consider migrating to an alternative over the coming
+  years. (#16152)
```

Comment:
I think this is not good advise.
BIP37, when connected to your own node, is the most efficient way to get a light wallet without the need of installing middlewares or the need of separate index incompatible with pruning.

For this reason, `whitebind` and `whitelist` now support the `bloomfilter` permission, so whitelisted peer can still enjoy BIP37.

Reply:
I'll add a mention of the bloomfilter permission (the ability to specify per-peer/network/interface options via white\* is already mentioned in the document).

That said, I'm skeptical of your argument.  For privacy, BIP37 needs to be over a secure private link (e.g. authenticated and encrypted), and if you have that, you can just run the wallet RPCs for even higher efficiency than BIP37 with minimal false positives.  Your pruning argument doesn't make sense to me (BIP37 needs full blocks, so it has problems with pruning that BIP158 doesn't---with BIP158, you can create the block filter and then throw away the full block and undo data); if you're thinking about Electrum-style indexes or Electrum Personal Server rescanning, BIP37 seems equivalent in problems related to pruning).  This is probably off-topic for release notes discussion, though, so DM me on IRC or poke me in #bitcoin if you're bored and want to debate it.  :-)

Reply:
DMed on twitter :)